### PR TITLE
glitter kittens!

### PR DESCRIPTION
**Pull request by ellamental**
First stab at implementing #2

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
-  intents: [ 
+const client = new Client({
+  intents: [
     GatewayIntentBits.Guilds,
-    GatewayIntentBits.GuildMembers, 
-    GatewayIntentBits.MessageContent 
-  ] 
+    GatewayIntentBits.GuildMembers,
+    GatewayIntentBits.MessageContent
+  ]
```
Comment (ellamental) [inline index.js]: Whoops, looks like my editor stripped some whitespace.  This shouldn't be part of this change.

File: index.js [hunk 2]
```diff
@@ -28,6 +28,9 @@ client.on('guildMemberAdd', (member) => {
   welcomeChannel.send(`Total Members: ${member.guild.memberCount}`);
 });
 
+// Event for when the glitter gets puked?
+client.on(/** code here */);
```
Comment (ellamental) [inline index.js:32]: IDK if I really understand what's going on here... I think this is probably not where we let the user hit the "puke glitter button", but maybe?

File: index.js (hunk 1 above)
Comment (MilitantMom) [reply index.js:7]: Got it! No worries, I'll remove that extra line and make sure everything is properly formatted. Thanks for catching that!

**Comment by MilitantMom** [comment]
I think I like the idea of /pg sending a random cat glitter gif, what do you think?
